pkg/policy/policy_peer.go

```diff
@@ -91,3 +93,64 @@ func (rc L4RuleContext) PortProto() string {
 	port := strconv.Itoa(int(byteorder.NetworkToHost(uint16(rc.Port)).(uint16)))
 	return port + "/" + proto
 }
+
+// EntityContexts maps am entity to a L4RuleContexts
+type EntityContexts map[api.Entity]L4RuleContexts
```

Comment:
I'm still struggling a bit with the name Context. I know that we came up with this together but these will become fundamental structures and it should be crystal clear.

Maybe just name it `EntityPolicy` so `Policy` will be a collection of different `xxxPolicy` types?